lib/config/screen.ex

```diff
@@ -122,14 +123,20 @@ defmodule ScreensConfig.Screen do
     |> String.ends_with?(@v2_app_id_suffix)
   end
 
-  for vendor <- ~w[gds mercury solari mimo c3ms outfront]a do
+  defp value_from_json("vendor", vendor_string, _app_id) when vendor_string in ["n/a", ""],
+    do: nil
```

Comment:
We probably want to include `nil` as a possible input here so the `null` that is ultimately saved to the configuration can round-trip successfully:

```elixir
defp value_from_json("vendor", vendor, _app_id) when vendor in [nil, "", "n/a"], do: nil
```